tests/e2e/framework/kubernetes.go

```diff
@@ -98,6 +100,8 @@ var (
 	caTag              = flag.String("ca_tag", os.Getenv("TAG"), "Ca tag")
 	galleyHub          = flag.String("galley_hub", os.Getenv("HUB"), "Galley hub")
 	galleyTag          = flag.String("galley_tag", os.Getenv("TAG"), "Galley tag")
+	cniHub             = flag.String("cni_hub", os.Getenv("ISTIO_CNI_HUB"), "CNI hub")
```

Comment:
why can't you just use HUB and TAG for this as well?

Reply:
I guess I could, but I thought since they are two different repos and different build process it is very possible you don't use the same HUB and TAG for both Istio and CNI.   For example if testing CNI changes you peg Istio to an official release.

Reply:
I'm not familiar with the work done so far on CNI.  What repo is it in? Should we integrate it into istio/istio and have it part of our standard build?

Reply:
@nmittler - Louis Ryan, Costin and others debated the repo structure we wanted when we first spawned the project.   It was decided it was best to have only a loose coupling of CNI to Istio.  It is handled a little like the Envoy (istio envoy that is) and the API repos are handled.  So there is not direct coupling between Istio and CNI,  but when CNI is desired you need the CNI images.